pkg/server/cluster_definition_handler.go

```diff
@@ -0,0 +1,95 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/url"
+)
+
+const authTokenRequestURL = "https://developers.redhat.com/auth/realms/rhd/protocol/openid-connect/token"
+const pullSecretRequestURL = "https://api.openshift.com/api/accounts_mgmt/v1/access_token"
+
+type ClusterDefinition struct {
+	ClusterName string `json:"clusterName"`
+	DNSDomain   string `json:"DNSDomain"`
+	Username    string `json:"username"`
+	Password    string `json:"password"`
+	PullSecret  string `json:"pullSecret"`
+}
+
+type DeveloperTokenResponse struct {
+	ExpiresIn        int    `json:"expires_in"`
+	RefreshExpiresIn int    `json:"refresh_expires_in"`
+	RefreshToken     string `json:"refresh_token"`
+	TokenType        string `json:"token_type"`
+	NotBeforePolicy  int    `json:"not-before-policy"`
```

Comment:
This should use underscores for consistency.

Reply:
This format comes from developer.redhat.com, I am not sure I can do anything about it.

Reply:
:cry: